base/Dockerfile

```diff
@@ -37,15 +37,28 @@ ENV LANG="C.UTF-8"
 # Set the classpath for JARs required by `cub`
 ENV CUB_CLASSPATH='"/usr/share/java/cp-base-new/*"'
 
+# Zulu openJDK
+ENV ZULU_OPENJDK="zulu8.40.0.25-ca-jdk8.0.222-linux_x64"
```

Comment:
i don't have a strong feeling about this, but maybe this should go in the common pom

Reply:
Do you think it should go to common-docker's pom or common's pom?